barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplemini.hpp

```diff
@@ -396,97 +397,6 @@ template <typename Curve> class ShpleminiVerifier_ {
 
         return { commitments, scalars, shplonk_evaluation_challenge };
     };
-    /**
-     * @brief Populates the vectors of commitments and scalars, and computes the evaluation of the batched
```

Comment:
these comments are still useful/relevant no? Should they be moved (possibly with tweaks) to `ClaimBatcher::update_batch_mul_inputs_and_batched_evaluation`?

Reply:
I tried to add clarifications in relevant places